microcosm_flask/tests/swagger/test_definitions.py

```diff
@@ -44,12 +44,12 @@ def test_build_swagger():
         operations = list(iter_endpoints(graph, match_function))
         swagger_schema = build_swagger(graph, ns, operations)
 
-    assert_that(swagger_schema, is_(equal_to({
```

Comment:
I changed the straight `equal_to` to nested `has_entries` to make finding issues on test failure easier.